404.html

```diff
@@ -4,9 +4,13 @@
 permalink: /404.html
 ---
 
-
-<div class="text-center">
-  <h1>404: Page not found</h1>
-  <p >Sorry, we've misplaced that URL or it's pointing to something that doesn't exist.  
-	<a href="{{ site.baseurl }}/">Head back home</a> to try finding it again.</p>
-</div>
+<div class = "row">
+                    <div class="col-md-6"style="margin:auto">
+                      <img src="./assets/img/new_trans_circular_logo.png" alt="Italian Trulli">
```

Comment:
What is an Italian Trulli?

Reply:
Also here it's best to use `{{ site.baseurl }}`
```suggestion
                      <img src="{{ site.baseurl }}/assets/img/new_trans_circular_logo.png" alt="Italian Trulli">
```

Reply:
It was a placeholder that I had used sorry about that. Wanted to make sure the tag was being used correctly.